pkt_trail_fe/apps/users/views.py

```diff
@@ -1,3 +1,53 @@
 from django.shortcuts import render
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+import requests
```

Comment:
The order of imports should be proper - This is just cosmetic but better to follow the convention 

1. 1st import all standard library modules (`os`) here
2. Then add an empty line
3. Import all third party modules `requests`, `django`, `rest_framework` etc
4. Then add an empty line
5. Then import any local package modules (right now None)